src/index.js

```diff
@@ -946,7 +952,19 @@ export function useDropzone(props = {}) {
           accept: acceptAttr,
           multiple,
           type: "file",
-          style: { display: "none" },
+          style: {
+            border: 0,
+            clip: 'rect(0, 0, 0, 0)',
+            'clip-path': 'inset(50%)',
+            height: '1px',
+            margin: '0 -1px -1px 0',
+            overflow: 'hidden',
+            padding: 0,
+            position: 'absolute',
+            width: '1px',
+            'white-space': 'nowrap',
+          },
+          'aria-placeholder': placeholder,
```

Comment:
This is a property that you can set with `getInputProps({'aria-placeholder': 'drop files'})`. Let's limit this change to just the style change.